README.md

```diff
@@ -72,7 +72,7 @@ cd Daemon/
 git submodule update --init --recursive
 ```
 
-If cmake complains about missing files in `recastnavigation/` folder or similar issue then you have skipped this step.
```

Comment:
The old wording seems more accurate - it checks for specific files, not a directory; I think the message is something like "Missing source file".

`libs/crunch/` would be a better example since Breakpad is off by default.

Reply:
OK, I can unify the other way. Good point for breakpad/crunch, I just picked the first one on alphabetical order, but crunch is better indeed.